frontend/src/employee-frontend/components/reports/TampereRegionalSurvey.tsx

```diff
@@ -0,0 +1,177 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import orderBy from 'lodash/orderBy'
+import range from 'lodash/range'
+import React, { useMemo, useState } from 'react'
+
+import { useTranslation } from 'employee-frontend/state/i18n'
+import { RegionalSurveyReportResult } from 'lib-common/generated/api-types/reports'
+import LocalDate from 'lib-common/local-date'
+import { constantQuery, useQueryResult } from 'lib-common/query'
+import Title from 'lib-components/atoms/Title'
+import { Button } from 'lib-components/atoms/buttons/Button'
+import ReturnButton from 'lib-components/atoms/buttons/ReturnButton'
+import Combobox from 'lib-components/atoms/dropdowns/Combobox'
+import Container, { ContentArea } from 'lib-components/layout/Container'
+import {
+  FixedSpaceColumn,
+  FixedSpaceRow
+} from 'lib-components/layout/flex-helpers'
+import { H2, Label } from 'lib-components/typography'
+import { Gap } from 'lib-components/white-space'
+
+import { renderResult } from '../async-rendering'
+import { FlexRow } from '../common/styled/containers'
+
+import ReportDownload from './ReportDownload'
+import { FilterLabel, FilterRow } from './common'
+import { tampereRegionalSurveyReport } from './queries'
+
+interface ReportQueryParams {
+  year: number
+}
+
+export default React.memo(function TampereRegionalSurveyReport() {
+  const { i18n } = useTranslation()
+  const t = i18n.reports.tampereRegionalSurvey
+  const today = LocalDate.todayInHelsinkiTz()
+
+  const [selectedYear, setSelectedYear] = useState<number | null>(
+    today.year - 1
+  )
+
+  const yearOptions = range(
+    LocalDate.todayInSystemTz().year,
+    LocalDate.todayInSystemTz().year - 4,
+    -1
+  )
+
+  const [activeParams, setActiveParams] = useState<ReportQueryParams | null>(
+    null
+  )
+
+  const emptyValue: RegionalSurveyReportResult = {
+    monthlyCounts: [],
+    year: 0
+  }
+
+  const reportResult = useQueryResult(
+    activeParams
+      ? tampereRegionalSurveyReport(activeParams)
+      : constantQuery(emptyValue)
+  )
+
+  const fetchResults = () => {
+    if (selectedYear) {
+      setActiveParams({
+        year: selectedYear
+      })
+    }
+  }
+
+  const sortedReportResult = useMemo(
+    () =>
+      reportResult.map((result) => {
+        return {
+          year: result.year,
+          monthlyCounts: orderBy(
+            result.monthlyCounts.map((mc) => ({
+              ...mc.municipalDaycareResults,
+              ...mc.familyDaycareResults,
+              ...mc.municipalShiftCareResults,
+              ...mc.assistanceResults
+            })),
+            [(i) => i.month]
```

Comment:
Tämän voisi tehdä myös näin:
```suggestion
            'month',
            'asc'
```